index.html

```diff
@@ -0,0 +1,222 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+</head>
+
+<body>
+    <section class="wrapper">
+        <nav class="toolbar">
+            <ul>
+                <li class="logo"><a href="#"><b>Basir Elham</b></a></li>
+                <li class="collapse-button"><a href="#"><span class="fa fa-list"></span></a></li>
+                <li class="menu">
+                    <span>
+                        <ul>
+                            <li><a href="#"><b>Portfolio</b></a></li>
+                            <li><a href="#"><b>About</b></a></li>
+                            <li><a href="#"><b>Contact</b></a></li>
+                        </ul>
+                    </span>
+                </li>
+            </ul>
+        </nav>
+        <div class="headline">
+            <div class="content">
+                <h1>Hey There <br> I'm Basir Elham</h1>
```

Comment:
I think your title is too big compared to what we have in the Figma design. Also, I don't think you need to wrap your title in a `b` tag again, since it is inside the `h1` tag. 
Kindly check the [Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122&t=4ACPn3aMfBRvXnfl-0) and adjust this.